benchmark/fft/fft_benchmark_gpu.cc

```diff
@@ -0,0 +1,109 @@
+#if TACHYON_CUDA
+#include <iostream>
+
+// clang-format off
+#include "benchmark/fft/fft_config.h"
+#include "benchmark/fft/fft_runner.h"
+#include "benchmark/fft/simple_fft_benchmark_reporter.h"
+// clang-format on
+#include "tachyon/c/math/polynomials/univariate/bn254_univariate_dense_polynomial_type_traits.h"
+#include "tachyon/c/math/polynomials/univariate/bn254_univariate_evaluation_domain_type_traits.h"
+#include "tachyon/c/math/polynomials/univariate/bn254_univariate_evaluations_type_traits.h"
+#include "tachyon/math/elliptic_curves/bn/bn254/fr.h"
+#include "tachyon/math/polynomials/univariate/univariate_evaluation_domain.h"
+#include "tachyon/math/polynomials/univariate/univariate_evaluation_domain_factory.h"
+
+namespace tachyon {
+
+using namespace math;
+
+template <typename Domain, typename PolyOrEvals,
+          typename RetPoly = std::conditional_t<
+              std::is_same_v<PolyOrEvals, typename Domain::Evals>,
+              typename Domain::DensePoly, typename Domain::Evals>>
+void Run(const FFTConfig& config) {
+  using F = typename Domain::Field;
+
+  std::string_view name;
+  if (config.run_ifft()) {
+    name = "IFFT Benchmark GPU";
+  } else {
+    name = "FFT Benchmark GPU";
+  }
+  SimpleFFTBenchmarkReporter reporter(name, config.exponents());
+  reporter.AddVendor("tachyon_gpu");
+
+  std::vector<uint64_t> degrees = config.GetDegrees();
```

Comment:
Requires modification according to this commit([refac(benchmark): change uint64_t to size_t for degree](https://github.com/kroma-network/tachyon/pull/466/commits/d99d47d5a79ad243d2da47a533d1e0c052fb71eb))

```suggestion
  std::vector<uint64_t> degrees = config.GetDegrees();
  std::vector<size_t> degrees = config.GetDegrees();
```